js/lesson-24/part8.js

```diff
@@ -0,0 +1,37 @@
+`use strict`;
+
+let $dayOfWeek;
+let $daysCounter;
+let $do = true;
+
+while ($do) {
+    for (let i = 0; i <= 6; i++) {
+        switch (i) {
```

Comment:
In `switch` statement you need to provide `default` case

Reply:
According to the specification default is an optional parameter.

Reply:
It is safer for developers to use `default`. Since you can catch and show errors or provide fallbacks. Your code should always be well described and flawless ^_^